Src/BootCamp.Chapter/Person.cs

```diff
@@ -0,0 +1,125 @@
+﻿using BootCamp.Chapter.Logger;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Person
+    {
+        private string _name;
+        private int _age;
+        private float _weight;
+        private float _height;
+
+        public void SetInformation(ILogger logger)
+        {
+            SetName(logger);
+            SetAge(logger);
+            SetWeight(logger);
+            SetHeight(logger);
+        }
+
+        public void SetName(ILogger logger)
+        {
+            while (true)
```

Comment:
quite scare to see such a loop condition. Try to design that access is given explicitly and not broken explicitly. Having it this way you risk an infinite loop more.

Reply:
funny, because I always thought so, but I've seen it A LOT in different python projects. And I just figured it's a standard.
Will  avoid it in future

Reply:
Because people don't care. People are result focused.

Subtle details like this don't require much effort to change, but significantly improve readability and maintainability.